packages/api/src/schema/schema.yaml

```
@@ -696,8 +695,9 @@ components:
       type: object
       additionalProperties: false
       properties:
```

Right! On this object you would declare 
```suggestion
      additionalProperties: false
      required: [response]
      properties:
```

To make the response field obligatory and remove the need to check manually on your API handler code. BUT I don't think it should be required anyway, just explaining here what you could do on another case like this (or if you prefer to keep it as required)